SwaprHQ/stackly-ui
# [SWA-198][INFRA] - Track click events on Landing page "Launch App" buttons

## Fixes: [SWA-198](https://linear.app/swaprhq/issue/STK-198/add-analytics-to-stackly-landing-page)

# Description
* Create Anaytics context, FA installation script. and `.env.example` file
* Update `ButtonLink` component and types to use `onClick` handlers
* Apply "Launch App" click tracking + site traffic analytics to the Landing page

# Visual Evidence
https://app.usefathom.com/?range=last_30_days&site=CRHSMTWK -> this is the link to Stackly dapp (dev env) site analytics

Screenshot: 
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/6c3814f5-39d7-4fd1-bd4f-1c3ff7458d08)



# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Stackly Landing 
4) Connect your wallet
5) Clicking any "Launch App" or "Stack Now" button should trigger a FA event
